modules/ROOT/pages/dataweave-formats-csv.adoc

```diff
@@ -167,71 +171,90 @@ The CSV streaming example produces the following output.
 ]
 ----
 
+// CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
 
-// CONFIG PROPS ///////////////////////////////////////////////////////
 
 [[properties]]
 == Configuration Properties
 
-DataWeave supports the following configuration properties for CSV.
+DataWeave supports the following configuration properties for this format.
 
+
+[[reader_properties]]
 === Reader Properties
 
-The CSV format accepts properties that provide instructions for reading input data.
+This format accepts properties that provide instructions for reading input data.
+
 
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-| `bodyStartLineNumber` | `Number` | `0` | The line number on which the body starts.
-| `escape` | `String` | `\` | Character used to escape invalid characters, such as separators or quotes within field values.
-| `header` | `Boolean` |`true` | Indicates whether a CSV header is present.
- Valid values are `true` or `false`.
-
-* If `header=true`, you can access the fields within the input
- by name, for example, `payload.userName`.
-* If `header=false`, you must access the fields by index, referencing
- the entry first and the field next, for example, `payload[107][2]`.
-| `headerLineNumber` | `Number` | `0` | The line number on which the CSV header is located.
-| `ignoreEmptyLine`| `Boolean` | `true` | Ignores any empty line.
+|Parameter |Type |Default|Description
+|`bodyStartLineNumber`  |`Number`|`0`|Line number on which the body starts.
+|`escape`  |`String`|`\`|Character to use for escaping special characters, such as separators or quotes.
+|`header`  |`Boolean`|`true`|Indicates whether a CSV header is present. 
+
+* If `header=true`, you can access the fields within the input by name, for example, `payload.userName`. 
+
+* If `header=false`, you must access the fields by index, referencing the entry first and the field next, for example, `payload[107][2]`.
+
+
+
+Valid values are `true` or `false`.
+|`headerLineNumber`  |`Number`|`0`|Line number on which the CSV header is located.
+|`ignoreEmptyLine`  |`Boolean`|`true`|Indicates whether to ignore an empty line.
+
+Valid values are `true` or `false`.
+|`quote`  |`String`|`"`|Character to use for quotes.
+|`separator`  |`String`|`,`|Character that separates one field from another field.
+|`streaming`  |`Boolean`|`false`|Streams input when set to `true`. Use only if entries are accessed sequentially. The input must be a top-level array.
```

Comment:
TODO: is streaming complete?

Reply:
I'm not sure what you mean by complete, but it has the limitation of only being able to access elements in a forward way (1, 2, 3), not random access (3, 1, 2). But in CSV it's almost always done in a forward only way, like `payload map {...}`

Reply:
```suggestion
|`streaming`  |`Boolean`|`false`|Streams input when set to `true`. Use only if entries are accessed sequentially and do not require random access. The input must be a top-level array. 
```